c7n/resources/dynamodb.py

```diff
@@ -146,8 +124,8 @@ def delete_table(self, client, table_set):
             client.delete_table(TableName=t['TableName'])
 
     def process(self, resources):
-        resources = self.filter_table_state(
-            resources, self.valid_status)
+        resources = self.filter_resources(
+            resources, 'TableStatus', self.valid_status)
```

Comment:
minor nit, we should be able to one line all of these, we switched from 80 col, to 100 col lines on flake8